### PR TITLE
Implemented Issue #97: retrieve all departments

### PR DESCRIPTION
**Pull request by KevinGus**
Added functionality "retrieveDepartments" in membersController.
Added API-route "/departments" in membersRoutes.
Added type in membersTypes.
Commented type.
Closes #99

### CODE REVIEW

**Review by christophtrybek — APPROVED** [review]
LGTM

File: server/src/members/membersTypes.ts [hunk 1]
```diff
@@ -42,4 +42,12 @@ export type GetDepartmentMembersQueryResult = {
   nachname: string;
   ressort: string;
   bezeichnung: string;
+};
```
Comment (LennartRukower) [inline server/src/members/membersTypes.ts:45]: Please add a new line between types.

**Review by LennartRukower — CHANGES_REQUESTED** [review]
Currently there are eight departments returned. The department with the id eight means "no department". When do we need the eight department?

**Review by christophtrybek — APPROVED** [review]
LGTM

**Review by LennartRukower — APPROVED** [review]
LGTM

**Pull request merged**